### ISSUE TITLE
Support for label selectors to choose pods to backup volumes using restic

### ISSUE THREAD

**Issue by ashish-amarnath** [vmware-tanzu/velero#2770]
Using label selection to get pods, return a list of all volumes attached to those pods and backup those volumes using restic

**Vote on this issue!**

This is an invitation to the Velero community to vote on issues, you can see the project's [top voted issues listed here](https://github.com/vmware-tanzu/velero/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc).  
Use the "reaction smiley face" up to the right of this comment to vote.

- :+1: for "The project would be better with this feature added"
- :-1: for "This feature will not enhance the project in a meaningful way"

**Comment by quentinbadoud** [comment]
I agree beacause with Velero 1.5 you can backup all pod volumes by default. Restic tries to backup EmptyDir volume and fails. Then you have partially failed backups even if they are OK.

**Comment by optimus-kart** [comment]
is there a way to exclude emptydir from backup/restore in 1.5?

**Comment by arsulegai** [comment]
Hi team, following up on this discussion thread. The feature of `--exclude-list` based on type of volume will be helpful. Code/deployment would be written to assume `emptyDir` would be lost during pod restart. There can be flexibility at the time of usage.
